src/Apps/Artist/Components/ArtistHeader/ArtistHeaderImage.tsx

```diff
@@ -17,6 +15,8 @@ export const ArtistHeaderImage: FC<ArtistHeaderImageProps> = ({
   image,
   ...rest
 }) => {
+  const lazyLoad = !(getENV("IS_GOOGLEBOT") || getENV("IS_MOBILE"))
```

Comment:
desktop LCP is passing, but mobile can improve, so don't lazy load the artist header image